Velaptor/OpenGL/GLWindow.cs

```diff
@@ -482,29 +490,33 @@ private void GLWindow_Render(double time)
         /// <param name="keyboard">The system keyboardInput.</param>
         /// <param name="key">The key that was pushed down.</param>
         /// <param name="arg3">Additional argument from OpenGL.</param>
-        private void GLKeyboardInput_KeyDown(IKeyboard keyboard, Key key, int arg3) => this.keyboardInput.SetState((KeyCode)key, true);
+        private void GLKeyboardInput_KeyDown(IKeyboard keyboard, Key key, int arg3)
+            => this.keyboardReactable.PushNotification(((KeyCode)key, true));
 
         /// <summary>
         /// Invoked when any keyboard input key transitions from the down position to the up position.
         /// </summary>
         /// <param name="keyboard">The system keyboardInput.</param>
         /// <param name="key">The key that was released.</param>
         /// <param name="arg3">Additional argument from OpenGL.</param>
-        private void GLKeyboardInput_KeyUp(IKeyboard keyboard, Key key, int arg3) => this.keyboardInput.SetState((KeyCode)key, false);
+        private void GLKeyboardInput_KeyUp(IKeyboard keyboard, Key key, int arg3)
+            => this.keyboardReactable.PushNotification(((KeyCode)key, false));
 
         /// <summary>
         /// Invoked when any of the mouse buttons are pressed into the down position over the window.
         /// </summary>
         /// <param name="mouse">The system mouse object.</param>
         /// <param name="button">The button that was pushed down.</param>
-        private void GLMouseInput_MouseDown(IMouse mouse, SilkMouseButton button) => this.mouseInput.SetState((VelaptorMouseButton)button, true);
+        private void GLMouseInput_MouseDown(IMouse mouse, SilkMouseButton button)
+            => this.mouseBtnReactable.PushNotification(((VelaptorMouseButton)button, true));
 
         /// <summary>
         /// Invoked when any of the mouse buttons are released from the down position into the up position over the window.
```

Comment:
What does "over the window" mean?  Is it needed?  Can you say, "...when the mouse is in focus in the window?"

Reply:
Come to me for simple clarification.  But here is an overview.
![image](https://user-images.githubusercontent.com/85414302/166874070-d4b8accd-18b2-4923-b1fe-2092a84351c0.png)